tests/test_trade_pipeline.py

```diff
@@ -1742,6 +1742,7 @@ def test_withdraw_reward_after_unilateral_cancel(self):
         )
         body = {
             "invoice": signed_payout_invoice,
+            "routing_budget_ppm": 0
```

Comment:
I think it's worth to leave this test as it is and create a new one specifically sending a custom number here